var/spack/repos/builtin/packages/py-currentscape/package.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+# replace all 'x-y' with 'xY' (e.g. 'Py-morph-tool' -> 'PyMorphTool')
+class PyCurrentscape(PythonPackage):
+    """Module to easily plot currentscape."""
+
+    homepage = "https://bbpcode.epfl.ch/code/#/admin/projects/cells/currentscape"
+    url = "ssh://bbpcode.epfl.ch/cells/currentscape"
+    git = "ssh://bbpcode.epfl.ch/cells/currentscape"
+
+    version("develop", branch="master")
+    version("0.0.1.dev0", tag="currentscape-v0.0.1.dev0", preferred=True)
```

Comment:
```suggestion
    version("0.0.1.dev0", tag="currentscape-v0.0.1.dev0")
```
`preferred` is only needed when tagging an older version to be the default one.